crates/chainspec/src/spec.rs

```diff
@@ -317,6 +317,10 @@ impl ChainSpec {
 
     /// Get the initial base fee of the genesis block.
     pub fn initial_base_fee(&self) -> Option<u64> {
+        if self.chain.named() == Chain::scroll_mainnet().named() {
+            return Some(0)
```

Comment:
I think this should be `None` no?

Reply:
fixed [4b92f77](https://github.com/scroll-tech/reth/pull/56/commits/4b92f778a77b8049510be02aac9ef35397b359f5)